src/App.tsx

```diff
@@ -15,48 +17,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_BY = {
+  ALPHA: 'alpha',
+  LENGTH: 'length',
+};
+
+type SortingParams = {
+  sortBy: string,
+  reverse: boolean
```

Comment:
[Tips on naming boolean variables](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
Fix all cases if present
```suggestion
  isReverse: boolean
```